spec/factories.rb

```diff
@@ -0,0 +1,8 @@
+# Post model factory
+FactoryBot.define do
+  factory :post do
+    title {"Title"}
+    description {"Description"}
+    user_id {1}
```

Comment:
šis būs liels No!

tev navn ekādas garantijas ka tev vispār būs useris ar tādu id kkur.

Jāizmanto asociacijas un objekti visur kur var.

Reply:
it just happens ka factory_bot_rails jau tev visu nodrošina out of the box:

aizstāj `user_id {1}` ar `user` un factory dinamiski buildos user objektu, vienīgi tev bus janodefinē arī user factory.

Reply:
Pagugle Factory_bot association examples